HW35_SlotMachineGame.py

```diff
@@ -0,0 +1,56 @@
+"""
+  Slot machine game
+"""
+class SlotMachineGame:
+
+    def __init__(self, balance: int, isMember = None, freeGame = True):
+        self.balance = balance
+        self.isMember = isMember
+        self.freeGame = freeGame
+
+    WIN = 1000000
+    GAME_FEE = 50
+
+    def _spin(self):
+        """
+                Game fee deducts from balance,
+                if two random numbers match it's a win, balance increases with the WIN amount
+        """
+        import random
```

Comment:
imports should be specified on the top of module, if we don't have special need for it like circular imports